index.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+const bodyParser = require('body-parser')
+const mongoose = require("mongoose")
+const cors = require("cors")
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(cors())
+
+const home = require("./routes/home.router.js")
```

Comment:
Import all your files before anything else and after that add code from `line: 6-9`